diff -uNr libgda-5.0.3.old/doc/C/libgda-5.0-docs.sgml libgda-5.0.3/doc/C/libgda-5.0-docs.sgml
--- libgda-5.0.3.old/doc/C/libgda-5.0-docs.sgml	2012-01-24 19:39:55.000000000 +0100
+++ libgda-5.0.3/doc/C/libgda-5.0-docs.sgml	2013-04-07 15:55:37.155192668 +0200
@@ -310,7 +310,6 @@
     <xi:include href="gettingstarted.xml"/>
     <xi:include href="migration.xml"/>
     <xi:include href="migration2.xml"/>
-    <xi:include href="migration3.xml"/>
     <xi:include href="prov-notes.xml"/>
     <xi:include href="limitations.xml"/>
   </part>  
diff -uNr libgda-5.0.3.old/tools/browser/doc/gda-browser-docs.sgml libgda-5.0.3/tools/browser/doc/gda-browser-docs.sgml
--- libgda-5.0.3.old/tools/browser/doc/gda-browser-docs.sgml	2012-01-24 19:06:49.000000000 +0100
+++ libgda-5.0.3/tools/browser/doc/gda-browser-docs.sgml	2013-04-07 15:30:00.193516920 +0200
@@ -161,7 +161,7 @@
 
   <part id="part_index">
     <title>Appendix</title>
-      <xi:include href="fdl-appendix.xml"/>
+      <xi:include href="fdl-appendix.sgml"/>
   </part>
 </book>
 
