--- libgda-5.2.0/configure.ac.orig	2013-11-15 17:06:38.881824566 +0100
+++ libgda-5.2.0/configure.ac	2013-11-15 17:30:10.915098425 +0100
@@ -260,6 +260,8 @@
 		if test "x$have_graphviz" = "xyes"
 		then
 			dnl test if new API is supported
+			save_CFLAGS="$CFLAGS"
+			CFLAGS="$CFLAGS $GRAPHVIZ_CFLAGS"
 			AC_MSG_CHECKING([whether Graphviz's new API is supported])
 			AC_COMPILE_IFELSE([AC_LANG_SOURCE([
 #include <gvc.h>
@@ -270,6 +272,7 @@
 }
 ])],
                         graphviz_new_api=yes, graphviz_new_api=no)
+			CFLAGS="$save_CFLAGS"
 
 			AC_MSG_RESULT($graphviz_new_api)
 			if test "$graphviz_new_api" = "yes"; then
