diff -urNp -x '*.orig' libgda-5.2.10.org/m4/java.m4 libgda-5.2.10/m4/java.m4
--- libgda-5.2.10.org/m4/java.m4	2021-10-06 22:22:24.497404419 +0200
+++ libgda-5.2.10/m4/java.m4	2021-10-06 22:22:25.204072831 +0200
@@ -183,6 +183,11 @@ m4_define([_JAVA_CHECK_INTERNAL],
 		    JTYPE="Sun JRE 11.0"
 		    JFLAGS="-Xlint:unchecked -Xlint:deprecation"
 		    ;;
+                JRE1.8.*)
+		    try_java=true
+		    JTYPE="Sun JRE 1.8"
+		    JFLAGS="-Xlint:unchecked -Xlint:deprecation"
+		    ;;
                 JREgcj-4*)
 		    try_java=true
 		    JTYPE="GCJ"
