--- libgda-5.2.0/libgda-ui/demos/ddl_queries.c.orig	2013-10-19 14:35:28.000000000 +0200
+++ libgda-5.2.0/libgda-ui/demos/ddl_queries.c	2013-11-15 17:05:25.348494222 +0100
@@ -478,7 +478,7 @@
 
 		dlg = gtk_message_dialog_new_with_markup (GTK_WINDOW (data->top_window),
 							  GTK_DIALOG_MODAL,
-							  msg_type, GTK_BUTTONS_CLOSE, msg);
+							  msg_type, GTK_BUTTONS_CLOSE, "%s", msg);
 		g_free (sql);
 		g_free (msg);
 
